ibm/data_source_ibm_pi_instance.go

```diff
@@ -141,6 +142,10 @@ func dataSourceIBMPIInstance() *schema.Resource {
 				Type:     schema.TypeInt,
 				Computed: true,
 			},
+			"placement_group_id": {
```

Comment:
Can re-use the constant here `PIPlacementGroupID`

Reply:
Done